app/views/admin/booths/_form.html.haml

```diff
@@ -0,0 +1,17 @@
+.row
+  .col-md-12
+    .page-header
+      %title Request a Booth
+.row
+  .col-md-8
+    = semantic_form_for(@booth, url: admin_conference_booths_path(@conference.short_title), html: {multipart: true}) do |f|
```

Comment:
Please check this path, keeping in mind that you are using this form both for new and edit actions.